Makefiles/card10.GNUmakefile

```diff
@@ -0,0 +1,44 @@
+ifneq (,$(findstring card10,$(DEFAULT_OUTMOD)))
+ifeq (,${FIRMWARE_HOME})
+	$(error Please set FIRMWARE_HOME)
+endif
+
+PLATFORM := freertos
+
+CC := arm-none-eabi-gcc
+AR := arm-none-eabi-ar
+LD := arm-none-eabi-ld
+STRIP := arm-none-eabi-strip
+
+# CFLAGS
+# various
+#CFLAGS += -mthumb
+#CFLAGS += -mcpu=cortex-m4
```

Comment:
Why is this commented out?

Reply:
it's in sledconf

Reply:
maybe we should make it add those for the card10 anyway, we know what the hardware looks like so it's fine.